fili-core/src/main/java/com/yahoo/bard/webservice/table/LogicalTable.java

```diff
@@ -18,13 +19,16 @@
 /**
  * A LogicalTable has a grain and a tablegroup of physical tables that satisfy the logical table.
  */
-public class LogicalTable extends Table implements Comparable<LogicalTable> {
+public class LogicalTable implements Table, Comparable<LogicalTable> {
 
     public static final String DEFAULT_CATEGORY = "General";
     public static final ReadablePeriod DEFAULT_RETENTION = Years.ONE;
 
     private TableGroup tableGroup;
+    private LogicalTableSchema schema;
+    private Granularity granularity;
```

Comment:
No good one.  Just failed to follow through on the logic of the changes.  Fixing.